rocketmq-docker/4.2.0/kubernetes/deployment.yaml

```diff
@@ -22,7 +22,7 @@ spec:
           - mountPath: /opt/store
             name: brokeroptstore
       - name: namesrv
-        image: apache/rocketmq-namesrv:4.2.0-dev
+        image: huanwei/rocketmq-namesrv:4.2.0-k8s
```

Comment:
@huanwei would you mind explaining the reason why you replace `apache` with `huanwei`(i.e. your name here)?

Reply:
@XiaoZYang Here the huanwei/rocketmq-namesrv:4.2.0-k8s is just tagged from apache/rocketmq-namesrv:4.2.0-k8s. You can find it in `play-kubernetes.sh`.

Reply:
@XiaoZYang Here for Kubernetes deployment I think we can also set the images as apache/xxx. And we leave the comments in `play-kubernetes.sh`. I will submit the update now.